examples/ffmpeg-react/src/App.tsx

```diff
@@ -0,0 +1,296 @@
+                            <div className="space-y-2">
+                                <span className="block text-md font-medium leading-6 text-white">
+                                    Output Video
+                                </span>
+                                <video
+                                    ref={outputVideoRef}
+                                    width="480"
+                                    controls
+                                    crossOrigin="anonymous"
+                                    className=" rounded-md shadow-sm"
+                                >
+                                    <source src={outputVideo.fileSrc} />
+                                    Your browser does not support the video tag.
+                                </video>
+                            </div>
+                        )}
+                </div>
+            ) : (
+                <div className="col-span-full">
+                    <label
+                        htmlFor="cover-photo"
+                        className="block text-sm font-medium leading-6 text-white"
+                    >
+                        Upload a video
+                    </label>
+                    <div
+                        {...getRootProps({ className: "" })}
+                        className="mt-2 flex justify-center rounded-lg border border-dashed border-white px-6 py-10"
+                    >
+                        <div className="text-center">
+                            <PhotoIcon
+                                className="mx-auto h-12 w-12 text-gray-200"
+                                aria-hidden="true"
+                            />
+                            <div className="mt-4 flex text-sm leading-6 text-white">
+                                <label
+                                    htmlFor="video-upload"
+                                    className="relative cursor-pointer rounded-md bg-white font-semibold text-indigo-600 focus-within:outline-none focus-within:ring-2 focus-within:ring-indigo-600 focus-within:ring-offset-2 hover:text-indigo-500"
+                                >
+                                    <span>Upload a file</span>
+                                    <input
+                                        id="video-upload"
+                                        name="video-upload"
+                                        type="file"
+                                        className="sr-only"
+                                        {...getInputProps()}
+                                    />
+                                </label>
+                                <p className="pl-1">or drag and drop</p>
+                            </div>
+                            <p className="text-xs leading-5 text-white/50">
+                                PNG, JPG, GIF up to 10MB
+                            </p>
```

Comment:
Asking to attach PNG, JPG or GIF doesn't make sense here (as it should be a video)